example/src/screens/Onramp/CryptoOnrampFlow.tsx

```diff
@@ -841,6 +870,37 @@ export default function CryptoOnrampFlow() {
             handleCollectCardPayment={handleCollectCardPayment}
             handleCollectBankAccountPayment={handleCollectBankAccountPayment}
           />
+
+          {selectedPaymentMethod === 'BankAccount' &&
+            currentPaymentDisplayData && (
+              <Collapse title="ACH Settlement Speed" initialExpanded={true}>
+                <View style={styles.segmentedRow}>
+                  <View style={styles.segment}>
+                    <Button
+                      title="Instant"
+                      variant={
+                        achSettlementSpeed === 'instant' ? 'primary' : 'default'
+                      }
+                      center
+                      onPress={() => setAchSettlementSpeed('instant')}
+                    />
+                  </View>
+                  <View style={{ width: 8 }} />
+                  <View style={styles.segment}>
+                    <Button
+                      title="Standard"
+                      variant={
+                        achSettlementSpeed === 'standard'
+                          ? 'primary'
+                          : 'default'
+                      }
+                      center
+                      onPress={() => setAchSettlementSpeed('standard')}
+                    />
+                  </View>
+                </View>
+              </Collapse>
+            )}
```

Comment:
Just an FYI that it looks like there's no longer a built-in RN segmented control component so I‘ve created a basic purpose-specific implementation rather than pulling in a 3rd party library. https://reactnative.dev/docs/segmentedcontrolios